roxie/ccd/ccdmain.cpp

```diff
@@ -1129,10 +1132,56 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                 if (port)
                 {
                     const char *protocol = roxieFarm.queryProp("@protocol");
+                    const char *passPhrase = nullptr;
```

Comment:
They are local, passed into createListener() just below.  It would take more code to call createListener() twice, once for ssl with these and once for non-ssl without them.